tests/e2e/controllers/messages.js

```diff
@@ -0,0 +1,208 @@
+const assert = require('chai').assert,
+  PouchDB = require('pouchdb'),
+  request = require('request-promise-native'),
+  Url = require('url'),
```

Comment:
IMO use lowercase for `Url` and `PouchDB` constants. You could argue for camelcase for `pouchDB` or `pouchDb` but I think lower is most obvious.